app/routes.py

```diff
@@ -74,6 +74,28 @@ async def add_registry(body: schemas.RegistryInput):
         ) from error
 
 
+@router.post(
```

Comment:
Instead of a POST, I would have used a DELETE passing the `registry_url` as URL parameter.

Reply:
@filopedraz I'll update it

Reply:
Resolve the conversation when handled :)